package.json

```diff
@@ -180,6 +180,7 @@
     "webtreemap-cdt": "^3.2.1"
   },
   "dependencies": {
+    "@paulirish/trace_engine": "^0.0.7",
```

Comment:
Doesn't really matter regardless so just out of curiosity, what's the bundle size impact? 😬